agents/gemini_data_analytics/intro_gemini_data_analytics_sdk.ipynb

```
@@ -0,0 +1,1558 @@
+        "    elif \"looker_explore_reference\" in datasource:\n",
+        "        source_name = format_looker_table_ref(\n",
+        "            getattr(datasource, \"looker_explore_reference\")\n",
+        "        )\n",
+        "    else:\n",
+        "        source_name = format_bq_table_ref(\n",
+        "            getattr(datasource, \"bigquery_table_reference\")\n",
+        "        )\n",
+        "\n",
+        "    print(source_name)\n",
+        "    display_schema(datasource.schema)\n",
+        "\n",
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "looker_client_secret"

(S105)